clients/cli/src/main/java/org/apache/gravitino/cli/commands/CreateTag.java

```diff
@@ -52,11 +52,17 @@ public CreateTag(
   /** Create tags. */
   @Override
   public void handle() {
-    boolean hasOnlyOneTag = tags.length == 1;
-    if (hasOnlyOneTag) {
-      handleOnlyOneTag();
-    } else {
-      handleMultipleTags();
+    int tagCount = (tags != null ? tags.length : 0);
+    switch (tagCount) {
+      case 0:
+        System.err.println(ErrorMessages.TAG_EMPTY);
+        break;
+      case 1:
+        handleOnlyOneTag();
+        break;
+      default:
+        handleMultipleTags();
+        break;
```

Comment:
I think an if/else block would be more readable here.

Reply:
As these are just tests you can add this to them `@SuppressWarnings("DefaultCharset")`

Reply:
I agree. A If/Else block is more appropriate in this case.